dmgroups.tex

```diff
@@ -135,29 +135,47 @@ \subsection{DM Leadership Team \label{sect:dmlt}}
 
 \subsection{DM Change Control Board \label{sect:dmccb}}
 
-The DMCCB has responsibility for issues similar to those of the LSST Change Control Board, but with its scope restricted to the DM Subsystem.
+The DMCCB has responsibility for issues similar to those of the LSST Change Control Board, but focused on the  DM Subsystem.
 The DMCCB reviews and approves changes to all baselines in the Subsystem, including proposed changes to the DM System Requirements (DMSR), reference design, sizing model, i.e. any LDM-series document.
-The Technical Baseline, including software/hardware and documentation, is written by DM and controlled by the DMCCB.
+The Technical Baseline, including software/hardware and documentation, is produced by DM and controlled by the DMCCB.
 DMCCB validates that the form and content of the Technical Baseline is consistent with LSST project standards such as the Systems Engineering Management Plan (SEMP) \citeds{LSE-17}.
 
 \begin{itemize}
-\item Charter/purpose
-	\begin{itemize}
-	\item Ensure that the DM Technical Baseline (LDM-xxx) documents are baselined and subsequently changed only when necessary and according to LSST and DM configuration control processes
-	\end{itemize}
-\item Membership
+\item Responsibilities:
 	\begin{itemize}
-	\item Chaired by the Systems Engineer (\secref{role:sysengineer}).
-	\item The DMCCB has the same membership as the Systems Engineering Team (\secref{sect:sysengt}).
-	\item For on-line virtual meetings, if a consensus or quorum or is not reached within one week, the DM Project Manager will make a unilateral decision
+	\item Determine when deliverables (controlled documents and software) are ready to be baselined (placed under configuration controlled status) or released. This include LDM series documents.
+	\item Review and approve/reject proposed changes to baselined items
+        \item Review and approves all RFCs prior to 'Adoption'.
+	\item Monitor and approve  DM software releases
+        \item Monitor DM Jira issues status.
+	\item Ensure that the DM  follows  LSST and DM configuration control processes.
 	\end{itemize}
-\item Responsibilities
+\item Membership:
 	\begin{itemize}
-	\item Determines when specification and deliverables are of sufficient maturity and quality to be baselined (placed under configuration controlled status) or released.
-	\item Reviews and approves/rejects proposed changes to baselined items
+        \item Core members:
+                \begin{itemize}
+                \item DM Project Manager
+                \item DM Subsystem Scientist
+	        \item Chaired by the Systems Engineer (\secref{role:sysengineer}).
```

Comment:
“Systems Engineer (Chair)”?

Reply:
Indentation throughout these nested itemizations is kinda funky; makes it harder to read the source.